app/models/itms/finished_project_activity_unit.rb

```diff
@@ -0,0 +1,5 @@
+class Itms::FinishedProjectActivityUnit < ApplicationRecord
+  self.table_name = 'itms.projekty_ukoncene_aktivity_subjekty'
```

Comment:
ano, je to odkaz na subjekty, ktory vsak obsahuje nejake informacie.

zistim dovod a pripadne ich zmazem.